### PR TITLE
[zh] Update at /doc/setup/_index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
about #34562

### CODE REVIEW

File: content/zh-cn/docs/setup/_index.md [hunk 1]
```diff
@@ -56,7 +56,23 @@ If you don't want to manage a Kubernetes cluster yourself, you could pick a mana
...
 bare metal environments.
 -->
-可以[下载 Kubernetes](/releases/download/)，在本地机器、云或你自己的数据中心上部署 Kubernetes 集群。
+你可以[下载 Kubernetes](/zh-cn/releases/download/)，在本地机器、云或你自己的数据中心上部署 Kubernetes 集群。
+
+<!-- 
+Several [Kubernetes components](/docs/concepts/overview/components/) such as `kube-apiserver` or `kube-proxy` can also be
+deployed as [container images](/releases/download/#container-images) within the cluster. 
+-->
+一些 [Kubernetes 组件](/zh-cn/docs/concepts/overview/components/)，如 `kube-apiserver` 或 `kube-proxy`，
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/_index.md:65]:
```suggestion
类似 `kube-apiserver` 或 `kube-proxy` 等一些 [Kubernetes 组件](/zh-cn/docs/concepts/overview/components/)
```

**Review by windsonsea — COMMENTED** [review]
pick nits

**Comment by howieyuen** [comment]
This page has some other changes to be resynced, (see: https://github.com/kubernetes/website/pull/34774).

File: content/zh-cn/docs/setup/_index.md (hunk 1 above)
Comment (howieyuen) [reply content/zh-cn/docs/setup/_index.md:65]: 这里我持保留意见，前者是用的状语，后者是用定语，读起来中文语义表达没差

Comment (windsonsea) [reply content/zh-cn/docs/setup/_index.md:65]: 嗯嗯，维持不变好了。主要是考虑到 en 可以碎片式的拼合，zh 这样就会显得直译和机翻。
Whatever, it is acceptable.

**Comment by tengqm** [comment]
Conflicts?

**Comment by tengqm** [comment]
Still working on this, @yanrongshi ?

**Pull request closed without merging**